README.md

```diff
@@ -107,80 +191,83 @@ Each session is 3 hours long
  * OpenStack architecture & example (Bruno)
  * Docker (Bruno)
- * Production platform explanation (Vincent)
+ * *Customer Production* platform explanation (Vincent)
  * OpenVPN setup
  * Waystation creation (see below)  --> pb need group defined.
  * Home work:
    * Continue to explore OpenStack from both UI & CLI
+   * Explore tools to use for the factory and automation systems
    * Create 9 groups of 5 students (one country per group), assign specialization (ops, devs, middleware, ... + backup), tool choice left to students, but licensing should be correct (prefer Open Source)
 
 ## Second session
 
- * Devops fundamentals (Christophe)
+ * DevOps fundamentals (Christophe)
  * Application overview (René)
  * Infrastructure as Code: OpenStack API as TD
  * Project explanation: Architecture of the use case - Specifications - Design Constraints & Goals (GitHub, automation, )
- * Home work: Continue to explore OpenStack API (Dev), Start Prod Infra setup
+ * Home work: Continue to explore OpenStack API (Dev), DevOps concept and tools, Start Prod Infra setup
 
 ## Third session
 
+* Team member role and responsibility. Assigning tasks... Who do what?
+    The customer like to see that...
+* Design and implement a flow to develop and deliver application architecture
 * Prod infra setup continued: Ability to launch a VM from an image using a heat template, attached to a network and a storage, and an object storage. private and public net are available and a floating IP attached to the VM.
 * Application architecture done: microservices identified, HA solved, Scalability solved. Design done (Paper work, UML ?).
 
 ## Fourth session
 
  * Prod infra setup finished: Ability to launch a VM from an image using a heat template, attached to a network and a storage, and an object storage. private and public net are available and a floating IP attached to the VM.
- * Application development ongoing
+ * Application architecture development ongoing
+ * Factory improvement to deliver fisrt version of the application ecosystem
 
 ## Fifth session
 
- * Prod infra setup review if needed
- * Application development ongoing
+ * Prod infra setup / factory review if needed
+ * Application ecosystem and software factory improved to deliver more.
 
 ## Sixth session
 
- * Prod infra setup review if needed
- * Application development done
+ * Application ecosystem continuous improvement.
 
 ## Seventh session
 
  * Synthesis and integration
  * Group Presentation
+ * Integrate a customer request (feature or bug fix) in the factory
 
 ## Eigth session
 
  * Evaluation of the projects
 
-# Evaluation and evaluation method
+# Evaluation
 
-Evaluation will be 15' per group and an additional 15' for the infra team.
+Evaluation will be 30' per group.
```

Comment:
For 9 groups that's 4.5 hours.

Reply:
That's what we did last year, and more... You want to reduce?